EE_API_automation/pytest/runGettingStarted.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+set -e
+# Do not reveal secrets
+set +x
+
+JENKINSLOG=$(pwd)/pytest_junit_logs.xml
+CLILOG=$(pwd)/pytest_cli_logs.log
+echo ""
+echo Using logfile $JENKINSLOG and $CLILOG
+echo ""
+if test ! $(which pytest)
+    then
+        echo "==> Check pre-requisites: python and pytest should be installed"
+else
+    echo "==> Running GettingStarted API tests"
+    if [ $# -lt 3 ]
+    then
+        echo "==> Insufficient arguments provided! Need minimum 3. Exiting."
+        exit
+
+    elif [ -z "$4" ] || [ "$4" = "False" ] || [ "$4" = "false" ]
```

Comment:
The case where only API tests are run is still missing:

elif [ "$4" = "True" ] || [ "$4" = "true" ]

Refer to the original run_me.sh

Reply:
done @rgarg1 added case where only API tests are run